### PR TITLE
[DOCFIX] Update FAQ.md of the Chinese version	

### PR DESCRIPTION
**Pull request by shizihao123**

### CODE REVIEW

File: docs/cn/FAQ.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+
+2. Alluxio可以运行在哪些平台和Java版本上?
+
+   Alluxio可以在Linux / Mac的各种发行版上使用Java 1.7或更高版本运行。
+
+3. 是否有简单的方法来查看群集的状态和运行状况？
+  
+   是的，一旦安装好Alluxio，可以通过访问`http://master_ip:19999`的UI界面来查看。
+
+4. 如果向集群添加新节点，为了平衡节点之间的磁盘空间利用率，Alluxio是否会进行重新平衡（即将缓存的块移动到新节点）？
```
Comment (ifcharming) [inline docs/cn/FAQ.md]: s/磁盘空间/内存空间？

feel free to update the English version as well

Comment (shizihao123) [inline docs/cn/FAQ.md]: Thank you for the review of the detail, and I will fix that soon. @ifcharming

Comment (shizihao123) [inline docs/cn/FAQ.md]: I have fixed FAQ.md both the [English version](https://github.com/Alluxio/alluxio/pull/4848) and the Chinese version. Please check again, many thanks to you. @ifcharming

File: docs/cn/FAQ.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+    [在此](https://alluxio.atlassian.net/browse/ALLUXIO-2532?jql=project%20%3D%20ALLUXIO%20AND%20status%20%3D%20Open%20AND%20labels%20%3D%20NewContributor%20AND%20assignee%20in%20(EMPTY))查看JIRA上的一些问题（标注给新贡献者的）。
+
+18. 在哪里可以报告问题或提出新功能？
+
+    我们使用[JIRA](https://alluxio.atlassian.net/projects/ALLUXIO)来跟进开发过程和相关问题。如果您想报告问题或提出新功能，请将其发布到JIRA上的Alluxio问题跟进器上。您需要在发布之前注册。
+
+19. 在哪里可以获取更多的帮助？
+
+    对于任何与安装、代码贡献和评价相关的问题，请发送邮件到[Alluxio 用户邮件列表](https://groups.google.com/forum/?fromgroups#!forum/alluxio-users)。
+    我们期待在那儿与您探讨。
```
Comment (jsimsa) [inline docs/cn/FAQ.md]: this file is missing newline character at the end of the file ... please fix

Comment (ifcharming) [inline docs/cn/FAQ.md]: https://github.com/Alluxio/alluxio/pull/4860

**Pull request merged**